encord/orm/model.py

```diff
@@ -39,6 +45,41 @@ class Model(base_orm.BaseORM):
     DB_FIELDS = OrderedDict([("model_operation", int), ("model_parameters", dict)])
 
 
+@dataclass
+class ModelConfiguration(Formatter):
+
+    model_uid: str
+    title: str
+    description: str
+    # The corresponding feature node hashes of the ontology object
+    feature_node_hashes: List[str]
+    model: AutomationModels
+
+    @classmethod
+    def from_dict(cls, json_dict: dict):
+        return ModelConfiguration(
+            model_uid=json_dict["model_hash"],
+            title=json_dict["title"],
+            description=json_dict["description"],
+            feature_node_hashes=cls._get_feature_node_hashes(json_dict["features"]),
+            model=cls._get_automation_model(json_dict["model"]),
+        )
```

Comment:
There is a distinction between model uids and model iteration uids (the model itself and different iterations of training). 
I think that these iteration ids should also be available through the `ModelConfguration`?
Otherwise, this functionality is useless if one wants to run inference, as the iteration uids are necessary for that operation.